src/backend/distributed/catalog/dependency.c

```diff
@@ -371,11 +440,54 @@ follow_order_object_address(void *context, const Form_pg_depend pg_depend)
  * assumed to be a (List **) to the location where all ObjectAddresses will be collected.
  */
 static void
-apply_add_to_target_list(void *context, const Form_pg_depend pg_depend)
+apply_add_to_target_list(void *context, Form_pg_depend pg_depend)
 {
 	List **targetList = (List **) context;
 	ObjectAddress *address = palloc0(sizeof(ObjectAddress));
 	ObjectAddressSet(*address, pg_depend->refclassid, pg_depend->refobjid);
 
 	*targetList = lappend(*targetList, address);
 }
+
+
+/*
+ * epxand_citus_supported_types base on supported types by citus we might want to expand
+ * the list of objects to visit in pg_depend.
+ *
+ * An example where we want to expand is for types. Their dependencies are not captured
+ * with an entry in pg_depend from their object address, but by the object address of the
+ * relation describing the type.
+ */
+static List *
+epxand_citus_supported_types(void *context, Form_pg_depend pg_depend)
```

Comment:
epxand -> expand.

Also, we don't seem to use `context`, so remove ?

Reply:
`context` is not used here in this expansion. For now this is the only expansion so the context is not used at all.

given it fits as pluggable function to `recurse_pg_depend` I think it is benificial to keep its signature the same. As with `follow` and `apply` the purpose of `context` is to control its behaviour from the initial call site of `recurse_pg_depend`.

If you feel strongly about it I can remove, but for consistency I think it is beneficial to keep `context` here.